core/src/main/scala/org/apache/spark/executor/TaskMetrics.scala

```diff
@@ -102,19 +102,20 @@ class TaskMetrics private[spark] () extends Serializable {
 
   // Setters and increment-ers
   private[spark] def setExecutorDeserializeTime(v: Long): Unit =
```

Comment:
Since these now all call add (and don't reset), should we consider perhaps renaming them (or calling reset before add)? They are internal so no harm in renaming.